ONSdigital/ssdc-rm-ddl
# SDCSRM-368 removing WCIS templates and resetting ddl

# Has the DDL schema changed?
*Pull requests which include schema changes should be labelled with the `schema change` label for visibility*

*The [common entity model pom.xml](ssdc-rm-common-entity-model/pom.xml) file `project.version` must be bumped
appropriately if there are any common entity code changes*

  version: 

# Motivation and Context

With WCIS finished and a new survey coming up. We want to remove all the old templates from our DDL and reset all the patches for the new survey. We'll be groundzeroing the DB as part of this work. 

# What has changed



- Removed sll the patches from the DDL
- Reset the ddl version
- Removed the WCIS templates
# How to test?



- Run the dev-build and make sure there's no WCIS templates leftover.
- Run the groundzero script and make sure the WCIS templates are gone in a GCP project.
- Run the patch_database.py script and make sure there's no patches to apply and it's at v1.0.0
# Links



[Jira](https://jira.ons.gov.uk/browse/SDCSRM-340)
# Screenshots (if appropriate):
